renn/data/datasets.py

```diff
@@ -3,11 +3,12 @@
 import tensorflow_datasets as tfds
 import tensorflow_text as text
 import tensorflow as tf
+tf.compat.v1.enable_eager_execution()
 
 import os
 
 from renn import utils
-from renn.data.tokenizers import load_tokenizer, SEP
+from renn.data.tokenizers import load_tokenizer, SEP, _punctuation_separator, _tensor_punctuation_separator
```

Comment:
Is `_punctuation_separator` used in this module? I don't see it

Reply:
Good catch, it isn't needed.

Reply:
Good catch, it wasn't being used.